README.md

```diff
@@ -152,3 +152,54 @@ Custom checks can be created as subclasses of `CheckerInterface` and should cont
 Where
 - `bool` is a True or False whether the check was successful
 - `str` is a message to display in the result JSON, typically `OK` or `Fail`
+
+## Translations
+Multiple language support is provided by `flask-babel`. Docs here: https://python-babel.github.io/flask-babel/#
+
+### Python Setup
+The python setup relies on 2 methods and an initialisation. In `create_app` we initialise `babel` on our app:
+
+    from fsd_utils.locale_selector.get_lang import get_lang
+
+    babel = Babel(flask_app)
+    babel.locale_selector_func = get_lang
+    LanguageSelector(flask_app)
+
+The `get_lang()` function reads the user-selected language from a cookie (if set by `LanguageSelector` - see below), or if that is not present uses a built-in function of babel to negotiate the language based on the request headers and the supported languages.
+
+`LanguageSelector` creates an additional route `/language/<language>` that sets the user's selected language in a cookie. Used the cookie rather than the session so it can be shared across the microservices.
+
+Set `SESSION_COOKIE_DOMAIN` on the app to the domain you want to set the cookie on.
```

Comment:
So on the authenticator we have a config var called "COOKIE_DOMAIN" which is what sets the dot-prefixed domain for multi-service cookies (like the fsd_auth_token cookie). Presumably this should be the same domain? Shall we conform the name of the key to the same thing (whichever)and maybe even set it to a default in commonconfig.py?

Reply:
Don't think we want to default this as it's going to vary per environment, but yeah we probably should use the same variable.